README.md

```diff
@@ -16,11 +16,10 @@ SocialConnect uses a federated model, meaning that anyone has the power to be an
 
 Here are some active issuers verifying and registering attestations:
 
-| Issuer Name | Address|
-|-----|-----|
-|Kaala| `0x6549aF2688e07907C1b821cA44d6d65872737f05`
-|Libera| `0x388612590F8cC6577F19c9b61811475Aa432CB44` (mainnet) `0xe3475047EF9F9231CD6fAe02B3cBc5148E8eB2c8` (alfajores)
-
+| Issuer Name | Address                                                                                                         |
+| ----------- | --------------------------------------------------------------------------------------------------------------- |
+| Kaala       | `0x6549aF2688e07907C1b821cA44d6d65872737f05`                                                                    |
```

Comment:
sorry know this isn't your change, but can we add if this is Kaala's mainnet/alfajores issuer address to be consistent with Libera?

Reply:
Kaala doesn't have one 😅

Reply:
oh no i mean can we specify either `(mainnet)` or `(alfajores)` for this issuer address like we do with Libera below?

Reply:
done